src/parquet/util/bit-stream-utils.inline.h

```diff
@@ -83,13 +85,9 @@ inline bool BitWriter::PutVlqInt(uint32_t v) {
   return result;
 }
 
-inline bool BitWriter::PutZigZagVlqInt(int32_t v) {
```

Comment:
Can you add these (zigzag) back? I think this makes the new decoders work.

You can leave a TODO for me to add tests.